### PR TITLE
Anonymous feedback 2.0

### PR DESCRIPTION
**Pull request by RennKat**
Application rebuild with an updated tech stack.
- React
- NodeJs
- ApolloGraphql

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  },
+  "keywords": [],
+  "contributors": [
+    "Ismail Orabi <ismail.orabi@yale.edu> (https://git.yale.edu/iao4)",
+    "Aliza Knight <aliza.knight@yale.edu> (https://git.yale.edu/ak2746)"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git@git.yale.edu:LMS/anonymous-feedback.git"
+  },
```
Comment (iorabi) [inline backend/package.json]: You can remove the `contributors` or update to our public github info.

Update repo link to public github

File: frontend/package.json [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  },
+  "keywords": [],
+  "contributors": [
+    "Ismail Orabi <ismail.orabi@yale.edu> (https://git.yale.edu/iao4)",
+    "Aliza Knight <aliza.knight@yale.edu> (https://git.yale.edu/ak2746)"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git@git.yale.edu:LMS/anonymous-feedback.git"
+  },
```
Comment (iorabi) [inline frontend/package.json]: Remove contributors or update to public github info

Update repository to public github

File: backend/package.json (hunk 1 above)
Comment (iorabi) [reply backend/package.json]: @RennKat Can you make the same change to the backend `package.json`

**Comment by RennKat** [comment]
Forgot the backend package.json. Fixed!

File: backend/package.json (hunk 1 above)
Comment (RennKat) [reply backend/package.json]: @iorabi Sorry about that - fixed!

**Review by iorabi — APPROVED** [review]

**Pull request merged**